spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/registry/DataFlowAppRegistryPopulator.java

```diff
@@ -27,34 +30,39 @@
  *
  * @author Mark Fisher
  */
-public class DataFlowUriRegistryPopulator implements InitializingBean {
+public class DataFlowAppRegistryPopulator implements InitializingBean, ResourceLoaderAware {
```

Comment:
After some discussion, we are going ahead merging this as is with the conclusion that `DataFlowAppRegistryPopulator` is only used by the tests.